cmake/find/FindGBenchmark.cmake

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2015-2016, University College London
```

Comment:
Okay. Alternatively we can fork upstream repository and do the fix (it will be faster). I can create [hunter-packages](https://github.com/hunter-packages) for you in this case.